### ISSUE TITLE
Crash with either this or Farseek when exploring

### ISSUE THREAD

**Issue by darloth** [delvr/Repose#9]
But I think it's this, so I've put the issue here.  When exploring, the game crashed a few times, and now crashes reliably on load.

Crash report is included, mod list should be attached, 

I have a save where it reliably crashes attempting to load.  Unfortunately I also have a lot of mods, so I'm not sure whether it's worth sending you that - let me know if you want it, or if there's anything else I can do to help you track it down.

Version is 1.10.2 though I suspect you can tell from the crash.  Let me know if anything else would be helpful to upload - this seems like it might be related to some of the other farseek crashes you have open issues for since it's on exploration only, though of course I can't be sure.

Here's the modlist:
[modlist.txt](https://github.com/delvr/Repose/files/835835/modlist.txt)

And here's the crash report.
[crash-2017-03-11_14.19.03-server.txt](https://github.com/delvr/Repose/files/835833/crash-2017-03-11_14.19.03-server.txt)

Edit: Removing Repose prevents the crash, as expected.  Will try turning it back on now I've got the chunk to generate, see if it happens again.  Edit2: Nope, now it is working, for now.  Want any more crash reports that look similar?

Edit3: Biome I was walking into was a BoP Meadow, though I suspect this will only matter if you decide it's a BoP problem.  I'll write it here so I don't forget.

**Comment by darloth** [comment]
Huh, managed to reproduce this in a slightly different location with Repose entirely disabled.  I guess that means it's not a problem here.  Must be something BoP related.  I'll post a new issue there, and close this one, but still if you think there's anything useful about it let me know.  I'll keep all the files around for a bit.

**Comment by delvr** [comment]
Thanks darloth, just let me know if the issue persists in Repose after being fixed on the other mod's end.

**Comment by darloth** [comment]
Alas, BoP is not being updated for 1.10, I checked and they have a policy that says don't even bother reporting it if it's not against 1.11.  I'm not running that right now so it's pretty useless to me.

I made a new world instead without BoP.  Sorry to take your time looking at this when it turns out Repose was just exposing an issue.  I think it would happen just about any time sand fell on a BoP plant in a certain state.  Not sure which, but I suspect sand into water plants is what did it.
